03_launch_mgmt_cluster.sh

```diff
@@ -113,8 +147,35 @@ function launch_cluster_api() {
     popd
 }
 
+#
+# Launch the cluster-api-controller manager (v1alpha2) in the system namespace.
+#
+
+function launch_core_cluster_api() {
+    pushd "${CAPIPATH}"
+      make generate
+      sed -i'' 's/capi-system/metal3/' config/default/kustomization.yaml
+      kustomize build config/default | kubectl apply -f -
+    popd
+}
+
+#
+# Launch the cluster-api-bootstrap-provider-kubeadm-controller manager (v1alpha2) in the metal3 namespace.
+#
+
+function launch_cluster_api_bootstrap_provider_kubeadm() {
+    pushd "${CABPKPATH}"
+      sed -i'' 's/cabpk-system/metal3/' config/default/kustomization.yaml
+      make deploy
+    popd
+}
+
 clone_repos
 sudo su -l -c 'minikube start' "${USER}"
 launch_baremetal_operator
 apply_bm_hosts
-launch_cluster_api
+launch_cluster_api_provider_baremetal
+if [ "${V1ALPHA2_SWITCH}" == true ]; then
```

Comment:
V1ALPHA2_SWITCH should maybe be defined in lib/common.sh to default to false ? Then it would not be needed to be defined in 04_verify.sh

Reply:
Done